.github/workflows/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,59 @@
+name: 🚀 Feature request
+description: Add contributors to guestbook 💡
+title: "Feature:"
+labels: [💡 feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+  - type: dropdown
+    attributes:
+      multiple: false
+      label: Type of feature
+      description: Select the type of feature request, the lowercase should also be the PR prefix.
+      options:
+        - "🍕 Feature"
+        - "🐛 Fix"
+        - "📝 Documentation"
+        - "🎨 Style"
+        - "🧑‍💻 Refactor"
+        - "🔥 Performance"
+        - "✅ Test"
+        - "🤖 Build"
+        - "🔁 CI"
+        - "📦 Chore"
+        - "⏩ Revert"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current behavior
+      description: Please describe the issue — for example, "As part of the Intro to Open Source course, we are encouraged to contribute to this repo.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Suggested solution
+      description: Suggest a solution — for example, "Add myself as a contributor."
+  - type: input
+    id: context
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem or helpful links here.
```

Comment:
This part is also not mentioned in the issue.

Reply:
Yeah I'm still figuring out how to fix it

Reply:
Unless you have another thought here, let's remove this part and we're good to go. 👍

Reply:
Done @adiati98